src/axom/quest/InOutOctree.hpp

```diff
@@ -177,9 +176,7 @@ class InOutOctree : public spin::SpatialOctree<DIM, InOutBlockData>
     setVertexWeldThreshold(DEFAULT_VERTEX_WELD_THRESHOLD);
   }
 
-  /**
-   * \brief Generate the spatial index over the surface mesh
-   */
+  /// \brief Generate the spatial index over the surface mesh
```

Comment:
Sorry for all the noise in this PR -- I used it as an opportunity to clean up documentation in the files related to the InOutOctree